packages/terra-notification-dialog/src/NotificationDialog.jsx

```diff
@@ -2,96 +2,81 @@ import React from 'react';
+   * The variant of notification to be rendered. This renders the dialog with the corresponding header and icon to the
+   * variant concept.
+   * Use one of `alert`, `error`, `warning`, `info`, or `custom`.
    */
-  header: PropTypes.string,
+  variant: PropTypes.oneOf(variants).isRequired,
   /**
-   * Title of the notification-dialog.
+   * The title to describe the high-level overview of why the notification-dialog is being displayed to the user.
    */
-  title: PropTypes.string,
+  dialogTitle: PropTypes.string,
   /**
-   * **Deprecated**, Message of the notification-dialog. Resolves to `startMessage`.
-   */
-  message: PropTypes.string,
-  /**
-   * Message to be displayed at the start of the notification dialog body.
+   * The text to provide more detail or defined terminology to be displayed at the start of the notification dialog body.
    */
   startMessage: PropTypes.string,
   /**
-   * Message to be displayed at the end of the notification dialog body.
+   * The text to provide more detail or defined terminology to be displayed at the end of the notification dialog body.
    */
   endMessage: PropTypes.string,
   /**
-   *  Non-plain text content to be inserted after `startMessage` and/or before `endMessage`
+   *  The content to be inserted after `startMessage` and/or before `endMessage` to provide more details to the user in the dialog body.
    */
   content: PropTypes.node,
   /**
-   * **Deprecated**, The Action of the primary button. Resolves to `acceptAction`
-   */
-  primaryAction: PropTypes.shape({
-    text: PropTypes.string,
-    onClick: PropTypes.func,
-  }),
-  /**
-   * The Action of the accept button.
+   * The button text and onclick values of the accept button.
    */
   acceptAction: PropTypes.shape({
     text: PropTypes.string,
     onClick: PropTypes.func,
   }),
   /**
-   * **Deprecated**, The Action of the secondary button. Resolves to `rejectAction`.
-   */
-  secondaryAction: PropTypes.shape({
-    text: PropTypes.string,
-    onClick: PropTypes.func,
-  }),
-  /**
-   * The Action of the reject button.
+   * The button text and onclick values of the reject button.
    */
   rejectAction: PropTypes.shape({
     text: PropTypes.string,
     onClick: PropTypes.func,
   }),
   /**
-   * The variant of notification to be rendered.
-   * Use one of `alert`, `error`, `warning`, `info`, `success`, `custom`.
+   * The pieces to populate a notification-dialog when `variant="custom"`.
    */
-  variant: PropTypes.oneOf([
-    variants.ALERT,
-    variants.ERROR,
-    variants.WARNING,
-    variants.INFO,
-    variants.SUCCESS,
-    variants.CUSTOM,
-  ]),
-  /**
-   * The icon to be used for a notification of type custom. This will not be used for any other notification types.
-   */
-  customIcon: PropTypes.element,
+  custom: PropTypes.shape({
+    /**
+     * The keyword used to represent & emphasis the intention of dialog message that that is being shown to the user.
```

Comment:
2 'that'.

Reply:
updated: https://github.com/cerner/terra-framework/pull/1105/commits/b3410aaa901b3b91b6f8d43cdaf01d4a4c133f79#diff-cb62dc7ce0197bc569a9952450bb9fe5R68